jupyter-docker/jupyter_notebook_config.py

```diff
@@ -10,4 +10,9 @@
 c.NotebookApp.ip = '*'
 c.NotebookApp.port = 8000
 c.NotebookApp.open_browser = False
-c.NotebookApp.token = ''
\ No newline at end of file
+c.NotebookApp.token = ''
+c.NotebookApp.alloworigin
```

Comment:
does this key need a value?

Reply:
Ahh, nice catch. Yeah it should be `c.NotebookApp.alloworigin = '*'`